Extension/package.json

```diff
@@ -43,6 +43,15 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "views": {
+      "references-view": [
+        {
+          "id": "CppReferencesView",
+          "name": "%c_cpp.contributes.views.cppReferencesView.title%",
+          "when": "cppReferenceTypes:hasResults && reference-list.hasResult"
```

Comment:
I think this should be ||, right?

Reply:
Or just cppReferenceTypes:hasResults ? If there isn't any reason to show the cppReferenceTypes if there are only confirmed references.

Reply:
Verified that reference-list.hasResult is also true if there are no confirmed references or if you cancel "find all references" while in progress.